docs/modules/objectives/blitz.mdx

```diff
@@ -27,6 +27,14 @@ Blitz maps should include `<respawn auto="true"/>` to prevent players from "hidi
   | `<join-filter>` | Filters when a player can join the match after it has started. | [Filter](/docs/modules/mechanics/filters) | `never` |
 </div>
 
+##### Blitz Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
+</div>
```

Comment:
```suggestion
| Attribute | Description | Value |
|---|---|---|
| `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |

```